IssueTriage.md

```diff
@@ -0,0 +1,103 @@
+# Triage Guidance for WebNN Repo
+
+TODO: Consider removing these, or fit these into the documentation below.
+
+- **enhancement** - unclear if this is about the spec text, the proposed API, etc. Prefer **feature request** etc.
```

Comment:
I did some further triage, and the issues with https://github.com/webmachinelearning/webnn/labels/enhancement are only a few. We should probably drop this label entirely.